config/routes.rb

```diff
@@ -37,7 +37,8 @@
       resource :reassign, only: [:new, :create]
       resource :return, only: [:new, :create]
       resource :complete, only: [:create]
-      resource :send_decisions, only: [:new, :create]
+      get 'what_do_you_want_to_do_next', to: 'send_decisions#new', as: :send_decisions
+      post 'what_do_you_want_to_do_next', to: 'send_decisions#create'
 
```

Comment:
I'll update the other path names once I have confirmed what they should be with UCD.